demos/profiling/profiling_container/requirements.txt

```diff
@@ -0,0 +1,26 @@
+# dev
+black
+flake8
+ipdb
+boto3
+
+norfair >= 0.3.0
+numpy
+opencv-python
+torch
+torchvision
+graphviz
+
+tqdm 
+cython 
+pycocotools
+
+# pytorch2trt:
+onnx
+onnxruntime
+
+pycuda
+tensorrt
+
+# download weights
```

Comment:
These should have fixed versions so the demo doesn't break in the future when updates to them get released